input/postgres/replication.go

```diff
@@ -76,12 +76,51 @@ SELECT client_addr,
 	FROM %s
  WHERE client_addr IS NOT NULL`
 
+const transactionIdSQLPg13 string = `
+SELECT
+	pg_current_xact_id(),
+	next_multixact_id
+FROM pg_catalog.pg_control_checkpoint()
+`
+
+const transactionIdSQLPg96 string = `
+SELECT
+	txid_current(),
+	next_multixact_id
+FROM pg_catalog.pg_control_checkpoint()
+`
+
+const transactionIdSQLDefault string = `
+SELECT
+	txid_current(),
+	1
+`
+
 func GetReplication(logger *util.Logger, db *sql.DB, postgresVersion state.PostgresVersion, systemType string) (state.PostgresReplication, error) {
 	var err error
 	var repl state.PostgresReplication
 	var sourceTable string
 	var replicationStandbySQL string
 	var replicationSQL string
+	var transactionIdSQL string
+
+	// Get Postgres Server stats first (even with Aurora, as long as it's not a replica)
```

Comment:
[note for reviewer]
Since `pg_current_xact_id()` can't be run in replicas (will return `ERROR:  cannot execute pg_current_xact_id() during recovery`), we should avoid running this if it's replica.
Previously, we were not collecting any replication info with Aurora, because of `Function pg_last_xlog_receive_location() is currently not supported for Aurora` error. However, there should be no problem getting the current xact info with Aurora, so not going to skip that part here (tested with Aurora that it works).

Reply:
This has moved to server stats